terraform/tags.tf

```diff
@@ -0,0 +1,11 @@
+module "beacons_label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
+  namespace   = "mca"
+  environment = terraform.workspace
```

Comment:
There is a variable of `env` defined in the `variables.tf`.

For consistency, is there any reason to not use that over the workspace value?

Reply:
I saw that.  Could we go with one or the other universally?  Unsure of why we use env variables in some cases and workspaces in others?

Reply:
Sure, I don't think we use `terraform.workspace` at all?

Reply:
Discussed on a call.  Decided to remove `env` from `variables.tf` in favour of using `terraform.workspace` to denote stage/environment to which to deploy.